openml/datasets/functions.py

```diff
@@ -1095,7 +1098,7 @@ def _get_dataset_parquet(
             openml._api_calls._download_minio_file(
                 source=cast(str, url), destination=output_file_path
             )
-        except (FileNotFoundError, urllib3.exceptions.MaxRetryError) as e:
+        except (FileNotFoundError, urllib3.exceptions.MaxRetryError, minio.error.ServerError) as e:
```

Comment:
We could opt to catch all exceptions here instead. It would "future proof" in case new kinds of errors are raised, though generally it is useful to know what the errors are early and evaluate if different mitigation strategies are appropriate.

Reply:
I agree that we should make sure to see new error messages.